lib/std/os/bits/linux.zig

```diff
@@ -204,6 +204,12 @@ pub const WEXITED = 4;
 pub const WCONTINUED = 8;
 pub const WNOWAIT = 0x1000000;
 
+// waitid id types
+pub const P_ALL = 0;
+pub const P_PID = 1;
+pub const P_PGID = 2;
+pub const P_PIDFD = 3;
```

Comment:
This looks like it should be an enum, e.g.
```suggestion
pub const P = enum(c_uint) {
    ALL = 0,
    PID = 1,
    PGID = 2,
    PIDFD = 3,
    _,
};
```

Reply:
Thanks @daurnimator, I've made that change